features/FxA-54-push-on-status-change/README.md

```diff
@@ -30,6 +30,9 @@ As a concrete metric
 we can measure the number of calls to `/recovery_email/status`
 and check that it decreases as this feature rolls out.
 
+We will also add additional metrics to signal when Desktop verifies 
+via Push.
```

Comment:
Good call, we definitely have a lot of slow verifiers that we don't want to drag the metrics down